extra/perror.c

```diff
@@ -330,11 +330,21 @@ int main(int argc,char *argv[])
                        (const uchar*) "Unknown Error", 13) &&
           (!unknown_error || strcmp(msg, unknown_error)))
       {
+#ifdef _AIX
+      // On AIX, unknow error return " Error <CODE> occurred."
```

Comment:
This looks like an expansion/alternative of the previous if condition right? If so combine with that.

spelling 'unknow[n]'.
Style  - remove space between `AIX_buffer[250]`. lower case variable names 
size seem excessive
`snprint(aix_buffer, sizeof(aix_buffer)....`
prevent static analysis tools complaining

Reply:
Is this OK for you; or do I need to add some `#ifdef _AIX` ?

```
for ( ; argc-- > 0 ; argv++)
    {

      found=0;
      code=atoi(*argv);
      msg = strerror(code);

      // On AIX, unknow error return " Error <CODE> occurred."
      char unknow_aix[30];
      snprintf(unknow_aix, sizeof(unknow_aix), " Error %3d occurred.", code);

      /*
        We don't print the OS error message if it is the same as the
        unknown_error message we retrieved above, or it starts with
        'Unknown Error' (without regard to case).
      */
      if (msg &&
          my_strnncoll(&my_charset_latin1, (const uchar*) msg, 13,
                       (const uchar*) "Unknown Error", 13) &&
          (!unknown_error || strcmp(msg, unknown_error)) &&
          !strcmp(msg, unknow_aix))
      {
        found= 1;
        if (verbose)
          printf("OS error code %3d:  %s\n", code, msg);
        else
          puts(msg);
      }
```

I cannot just adapt `msg` because it is defined before we know the error code and AIX error message has error code inside.